{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# PDFs with python 2.7\n",
    "\n",
    "This is a documentation about **how to extract information from PDFs using python 2.7**.\n",
    "\n",
    "Date: 13 Dec (2017)\n",
    "\n",
    "\n",
    "I needed to record this process because it was difficult to find a good package which let me extract text from any PDF file. Maybe I was expecting too much from them. \n",
    "\n",
    "First, I needed to understand how the pdf file format works.\n",
    "\n",
    "In quora, I found the next [comment](https://www.quora.com/Is-there-an-easy-to-use-Python-library-to-read-a-PDF-file-and-extract-its-text): \n",
    "\n",
    "    We all expect to be able to use a library to parse some file format for text and be able to iterate through the text line by line, but what if the text has no line characters?\n",
    "\n",
    "    How would the library know what constitutes a line? Most libraries won’t try to guess at that, and honestly we wouldn’t want them to, because if the line isn’t represented by a line character, then the concept of line isn’t really part of the text (is it?) and we are using the library to extract *text*.\n",
    "\n",
    "    In pdf, text is laid out, meaning that a particular text object get displayed at a particular x,y position on the page. So what you might think of as 3 lines would actually be 3 text objects, displayed at (x,y), (x, y-20), (x, y-40) respectively, so a text extraction library would just pull out the text, but you’d have no line data. (IRRC pdfminer hands you String as output, just a big String, not a (line) iterable, it was because PDFMiner didn’t work for me that I had to study up and learn a bit about pdf to get what I wanted out of the files).\n",
    "\n",
    "    The upside is this — You finally get a chance to ‘roll your own.’\n",
    "\n",
    "    Fortunately, extracting the text out of a pdf is very well defined and simple goal. And fortuanately, PDF is a very well documented and very well understood file format, so google is going to be very helpful. If push comes to shove, the text rendering part of the spec is less than 200 pages, but you won’t need to go there.\n",
    "\n",
    "    Start here:\n",
    "\n",
    "    https://web.archive.org/web/20141010035745/http://gnupdf.org/Introduction_to_PDF\n",
    "\n",
    "\n",
    "    Then read the wikipedia article which is super well written. Then you will have to open the file in text editor and study it, which won’t be hard if you are interested only in text. Use this as a tool to understand the stream writing operators: \n",
    "\n",
    "    http://www.verypdf.com/document/pdf-format-reference/pg_0985.htm\n",
    "\n",
    "    The accepted answer to the following SO tells you what you need to investigate to understand how text is encoded within the pdf:\n",
    "\n",
    "    https://stackoverflow.com/questions/4047953/programatically-rip-text-from-a-pdf-file-by-hand-missing-some-text\n",
    "\n",
    "    Google anything you wish to understand, and you will be brought to cool sites like planetpdf, where they have great articles.\n",
    "\n",
    "    It should take you a day or two to hand write your parser and you will learn a lot in the process about something pretty common. The libraries have to be general, so they are going to be limited.\n",
    "\n",
    "    (Perhaps irrelevant, the pdfs I was working with are linearized—see the linked references—which made studying the text in the pdf and mapping to the layout on the screen super simple, I didn’t study an non-linearized files because i didn’t have to, but if it makes things harder there’s a ton of code out there to linearize a pdf but not a lot out there that can go the otherway)\n",
    "    \n",
    "   **Naftali Michalowsky**\n",
    "    \n",
    "***\n",
    "Another comment I found in the same page said:\n",
    "\n",
    "    This may come to you as a surprise, but PDF was never actually intended as a format for easy text extraction. Indeed, its primary purpose is to make sure that whatever is in the document would be displayed in a consistent manner across multiple platforms, as well as print identically (as much as that is possible) everywhere. This is achieved in many different ways, and sometimes goes as far as literally saying \"this letter goes here, this letter goes here, and this letter goes here\" in its internal language.\n",
    "\n",
    "    So yes, the libraries to extract text from PDF are insanely complicated by necessity, and they sometimes do not even function properly, because PDF is not meant for that.\n",
    "    \n",
    "   **Paulina Jonušaitė**\n",
    "\n",
    "***\n",
    "\n",
    "So I decided to take some packages and take a look of their performance \"reading\" PDF files."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## PDF Samples\n",
    "\n",
    "In order to explore the tool's functionalities, I have downloaded three pdfs.  \n",
    "\n",
    "One of them named <font color=\"red\">sample.pdf</font> which was obtained from this tutorial: http://programacion.net/articulo/como_trabajar_con_documentos_pdf_utilizando_python_1398.  \n",
    "\n",
    "Other one named <font color=\"red\">pdf-sample.pdf</font> which was obtained from: http://unec.edu.az/application/uploads/2014/12/pdf-sample.pdf.  \n",
    "\n",
    "Last one is a paper.\n",
    "\n",
    "I have created one directory named **pdfs** to contain the documents. My work directory looks like:\n",
    "\n",
    "\n",
    "    PDF_with_Python/\n",
    "    ├── LICENSE\n",
    "    ├── pdfs\n",
    "    │   ├── paper.pdf\n",
    "    │   ├── pdf-sample.pdf\n",
    "    │   └── sample.pdf\n",
    "    ├── PDF_with_Python.ipynb\n",
    "    └── README.md"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Tools information\n",
    "\n",
    "There exists different python tools for manipulating PDFs. Our interest in this notebook is to address this issue using **python 2.7**. Next tools are reported to work with python 2.\n",
    "\n",
    "**But remember that _Python 2.7_ will not be maintained past 2020**.\n",
    "\n",
    "### [PDFMiner](http://euske.github.io/pdfminer/index.html \"PDFMiner\")\n",
    "- Last Modified: Fri Mar 28 09:17:06 UTC 2014.  \n",
    "- **Python 2.x** only.  \n",
    "- MIT License.  \n",
    "- To install [this package](https://anaconda.org/conda-forge/pdfminer \"pdfminer conda\") with [conda](https://anaconda.org/ \"anaconda cloud\") run:\n",
    "    \n",
    "        conda install -c conda-forge pdfminer\n",
    "        \n",
    "- When I did using conda, the scripts are not globally recognized. So I had to install following next steps:\n",
    "    + Install Python 2.4 or newer. (Python 3 is not supported.)\n",
    "    + Download the PDFMiner source: https://pypi.python.org/pypi/pdfminer/\n",
    "    + Unpack it.\n",
    "    + Run setup.py to install:\n",
    "         \n",
    "            sudo python setup.py install\n",
    "\n",
    "    + Do the following test:\n",
    "\n",
    "            pdf2txt.py samples/simple1.pdf\n",
    "            \n",
    "            Hello\n",
    "\n",
    "            World\n",
    "\n",
    "            Hello\n",
    "            \n",
    "            World\n",
    "\n",
    "            H e l l o\n",
    "\n",
    "            W o r l d\n",
    "\n",
    "            H e l l o\n",
    "\n",
    "            W o r l d\n",
    "    + Done!\n",
    "\n",
    "PDFMiner is a tool for extracting information from PDF documents. Unlike other PDF-related tools, it focuses entirely on getting and analyzing text data.\n",
    "\n",
    "PDFMiner allows one to obtain the exact location of text in a page, as well as other information such as fonts or lines. It includes a PDF converter that can transform PDF files into other text formats (such as HTML). It has an extensible PDF parser that can be used for other purposes than text analysis.\n",
    "\n",
    "#### Tutorials\n",
    "\n",
    "- [PDFMiner API Doc](http://unixuser.org/~euske/python/pdfminer/programming.html)\n",
    "- [PDFMiner tutorial 1](http://denis.papathanasiou.org/posts/2010.08.04.post.html)\n",
    "- [PDFMiner tutorial 2](https://www.binpress.com/tutorial/manipulating-pdfs-with-python/167)\n",
    "- [PDFMiner tutorial 3](https://wiki.carleton.edu/display/itskb/PDFMiner%3A+Extracting+Text+from+a+PDF+File)\n",
    "\n",
    "#### Features\n",
    "\n",
    "- Written entirely in Python. (for version 2.4 or newer)\n",
    "- Parse, analyze, and convert PDF documents.\n",
    "- PDF-1.7 specification support. (well, almost)\n",
    "- CJK languages and vertical writing scripts support.\n",
    "- Various font types (Type1, TrueType, Type3, and CID) support.\n",
    "- Basic encryption (RC4) support.\n",
    "- PDF to HTML conversion (with a sample converter web app).\n",
    "- Outline (TOC) extraction.\n",
    "- Tagged contents extraction.\n",
    "- Reconstruct the original layout by grouping text chunks.\n",
    "        \n",
    "Though PDFminer is simple it has its own DRAWBACKS:\n",
    "\n",
    "   + Getting simple things done, like extracting the text is quite complex. The program is not designed to return Python objects, which makes interfacing things irritating.\n",
    "   + It’s an extremely complete set of tools, with multiple and moderately steep learning curves.\n",
    "   + It’s not written with hackability in mind.\n",
    "\n",
    "### [Textract](http://textract.readthedocs.io/en/latest/)\n",
    "- Latest commit on 21 Jul (2017).  \n",
    "- Requires **PDFMiner**, pdf2text and other solutions.  \n",
    "- **Python 2**.  \n",
    "- It is supposed to work in **Python 3** too using **pdfminer.six**, but when I tried to install it with conda (for python 3) it generated conflicts.\n",
    "- Unspecified License.  \n",
    "- Includes **command line interface**:\n",
    "        \n",
    "        textract path/to/file.extension\n",
    "        \n",
    "- Project's **GitHub**: https://github.com/deanmalmgren/textract \n",
    "- **Documentation**: http://textract.readthedocs.io/en/latest/   \n",
    "- **Documentation for Python Package**: http://textract.readthedocs.io/en/latest/python_package.html\n",
    "- Current versión (up to date): **1.6.1** (manually) \n",
    "- To install [this package](https://anaconda.org/conda-forge/textract \"Textract conda\") with [conda](https://anaconda.org/ \"anaconda cloud\") run one of the following (version 1.5.0):\n",
    "\n",
    "        conda install -c conda-forge textract \n",
    "\n",
    "### [PdfQuery](https://github.com/jcushman/pdfquery \"PdfQuery\")\n",
    "- Latest commit on 29 Jul (2017).  \n",
    "- Requires **PDFMiner**, pyquery and lxml libraries.  \n",
    "- Includes sample code, documentation.  \n",
    "- Seems to be **Python 2.x**. It is not clear if this tool is Python 2.x or Python 3.x.  \n",
    "- MIT License.  \n",
    "- To install [this package](https://anaconda.org/jacksongs/pdfquery \"PdfQuery conda\") with [conda](https://anaconda.org/ \"anaconda cloud\") run:\n",
    "    \n",
    "        conda install -c jacksongs pdfquery"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Textract\n",
    "\n",
    "First we have to import the module"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import textract"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Now let's try to read a document. \n",
    "\n",
    "***\n",
    "Let's try to read <font color=\"red\">sample.pdf</font>:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "text = textract.process(\"./pdfs/sample.pdf\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "What happened? ... Let's print **text** content:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "This is a sample PDF document Iâ€™m using to follow along with the tutorial\n",
      "\n",
      "\f",
      "\n"
     ]
    }
   ],
   "source": [
    "print(text)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "It seems pretty good.\n",
    "\n",
    "***\n",
    "Let's try with the file <font color=\"red\">pdf-sample.pdf</font>:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Adobe Acrobat PDF Files\n",
      "Adobe® Portable Document Format (PDF) is a universal file format that preserves all\n",
      "of the fonts, formatting, colours and graphics of any source document, regardless of\n",
      "the application and platform used to create it.\n",
      "Adobe PDF is an ideal format for electronic document distribution as it overcomes the\n",
      "problems commonly encountered with electronic file sharing.\n",
      "•\n",
      "\n",
      "Anyone, anywhere can open a PDF file. All you need is the free Adobe Acrobat\n",
      "Reader. Recipients of other file formats sometimes can't open files because they\n",
      "don't have the applications used to create the documents.\n",
      "\n",
      "•\n",
      "\n",
      "PDF files always print correctly on any printing device.\n",
      "\n",
      "•\n",
      "\n",
      "PDF files always display exactly as created, regardless of fonts, software, and\n",
      "operating systems. Fonts, and graphics are not lost due to platform, software, and\n",
      "version incompatibilities.\n",
      "\n",
      "•\n",
      "\n",
      "The free Acrobat Reader is easy to download and can be freely distributed by\n",
      "anyone.\n",
      "\n",
      "•\n",
      "\n",
      "Compact PDF files are smaller than their source files and download a\n",
      "page at a time for fast display on the Web.\n",
      "\n",
      "\f",
      "\n"
     ]
    }
   ],
   "source": [
    "text = textract.process(\"./pdfs/pdf-sample.pdf\")\n",
    "print(text)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Last result was so good although it seems disordered. \n",
    "\n",
    "***\n",
    "What happen when we try to read more complex PDFs like the **paper** we have downloaded?\n",
    "\n",
    "Let's try with the file <font color=\"red\">paper.pdf</font>:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Leukemia Research 34 (2010) 677–681\n",
      "\n",
      "Contents lists available at ScienceDirect\n",
      "\n",
      "Leukemia Research\n",
      "journal homepage: www.elsevier.com/locate/leukres\n",
      "\n",
      "Brief communication\n",
      "\n",
      "Micro-RNA-15a and micro-RNA-16 expression and chromosome 13 deletions in\n",
      "multiple myeloma\n",
      "Sophie L. Corthals a , Mojca Jongen-Lavrencic a , Yvonne de Knegt a , Justine K. Peeters a ,\n",
      "H. Berna Beverloo b , Henk M. Lokhorst c , Pieter Sonneveld a,∗\n",
      "a\n",
      "b\n",
      "c\n",
      "\n",
      "Department of Hematology, Erasmus Medical Centre Rotterdam, Rotterdam, The Netherlands\n",
      "Department of Clinical Genetics, Erasmus Medical Centre Rotterdam, Rotterdam, The Netherlands\n",
      "Department of Hematology, University Medical Center Utrecht, Utrecht, The Netherlands\n",
      "\n",
      "a r t i c l e\n",
      "\n",
      "i n f o\n",
      "\n",
      "Article history:\n",
      "Received 15 October 2009\n",
      "Received in revised form 28 October 2009\n",
      "Accepted 28 October 2009\n",
      "Available online 23 December 2009\n",
      "Keywords:\n",
      "Multiple myeloma\n",
      "Micro-RNA\n",
      "Chromosome 13 deletion\n",
      "\n",
      "a b s t r a c t\n",
      "We have used copy number variation (CNV) analysis with SNP mapping arrays for miRNA-15a and\n",
      "miRNA-16-1 expression analysis in patients with multiple myeloma (MM) with or without deletion of\n",
      "chromosome 13q14. MiRNA-15a and miRNA-16 display a range of expression patterns in MM patients,\n",
      "independent of the chromosome 13 status. These ﬁndings suggest that genes other than miR-15a and\n",
      "miR-16 may explain the prognostic signiﬁcance of 13q14 deletions.\n",
      "© 2009 Elsevier Ltd. All rights reserved.\n",
      "\n",
      "1. Introduction\n",
      "Multiple myeloma (MM) is a dismal disease with a median survival of 3–5 years [1].\n",
      "MM is characterized by genetic instability and chromosomal\n",
      "translocations early in disease progression play an important role\n",
      "in the pathogenesis of MM. Chromosome 13 aberrations occur in\n",
      "50% and the most commonly deleted marker in the 13q14 region is\n",
      "D13S319, located distal to the retinoblastoma-1 (RB1) gene [2,3]. It\n",
      "is now common practice to determine the presence of chromosome\n",
      "13 deletions by ﬂuorescence in situ hybridization (FISH) [4–10].\n",
      "The adverse prognostic role of these deletions has been attributed\n",
      "to the presence of MM tumor suppressor gene(s) on chromosome\n",
      "13q which as yet have not been identiﬁed.\n",
      "Micro-RNAs (miRs) are a class of small non-coding single\n",
      "stranded RNAs of approximately 22 nucleotides in length [11]. MiRs\n",
      "negatively regulate gene expression by binding to partially complementary sites in messenger RNAs (mRNAs) [12]. miRs may play\n",
      "a role in the pathogenesis of MM [13,14]. In chronic lymphocytic\n",
      "leukemia (CLL), miR-15a and miR-16-1 are frequently downregu-\n",
      "\n",
      "∗ Corresponding author at: Department of Hematology, Erasmus Medical Centre\n",
      "Rotterdam, Room No. L439, PO Box 2040, 3000 CA Rotterdam, The Netherlands.\n",
      "Tel.: +31 10 7033123; fax: +31 10 7035814.\n",
      "E-mail address: p.sonneveld@erasmusmc.nl (P. Sonneveld).\n",
      "0145-2126/$ – see front matter © 2009 Elsevier Ltd. All rights reserved.\n",
      "doi:10.1016/j.leukres.2009.10.026\n",
      "\n",
      "lated [15]. We evaluated the expression of miR-15a and miR-16-1\n",
      "in MM.\n",
      "2. Materials and methods\n",
      "We obtained bone marrow samples from newly diagnosed patients with MM.\n",
      "Myeloma plasma cells were puriﬁed >80% using CD138 magnetic microbeads\n",
      "(MACS system, Miltenyi Biotec, Bergisch Gladbach, Germany). Small RNA transcripts less than 200 nucleotides in length including miRs, were isolated from\n",
      "CD138 magnetic cell selected (MACS) MM plasma cells using miRVanaTM miRNA\n",
      "Isolation Kit (Ambion, Austin, TX). RNA levels and quality were assessed using\n",
      "the Agilent 2100 Bioanalyzer (Agilent Technologies). Mature miR-15a and miR16 expression levels in MM plasma cells were determined by real-time PCR\n",
      "using TaqMan miR assay (Applied Biosystems, P/N: 4373123 and 4373121)\n",
      "[16]. All experiments were performed in duplicate and miRs were considered\n",
      "as present when CT -values were lower than 35. Genomic DNA was isolated\n",
      "from peripheral blood by salting out precipitation. Tumor DNA was extracted\n",
      "from CD138 MACS MM plasma cells using Qiagen RNAeasy kit (Qiagen, Valencia, CA). Genome-wide SNP genotyping was performed using the Illumina\n",
      "Inﬁnium HumanHap550 Genotyping BeadChip, containing over 550,000 unique\n",
      "tag SNP markers, according to manufacturer’s instructions (Illumina, San Diego,\n",
      "CA).\n",
      "Genotypes for all arrays were calculated using BeadStudio’s genotyping module (v2.0, Illumina). Data was imported into Partek Genomics Suite 6.4 software\n",
      "(Partek Inc., St Louis, MO) for further analysis; allele intensities were calculated\n",
      "for 25 genomic DNA MM samples and 26 tumor DNA MM samples, of which 20\n",
      "had paired normal samples. For these 20 samples, paired analysis was performed.\n",
      "Allele speciﬁc copy number variation (AsCN) was calculated by estimating the number of copies for each allele, rather than total copies of each chromosome. Loss of\n",
      "heterozygosity (LOH) was estimated from the imported sample genotypes by a Hidden Markov Model, using the same baseline as in the segmentation algorithm (25\n",
      "\n",
      "\f",
      "678\n",
      "\n",
      "Table 1\n",
      "Chromosome 13 aberrations of the region ﬂanking miR-15a and miR-16-1 identiﬁed by FISH and segmentation analysis using SNP mapping arrays.\n",
      "FISH\n",
      "\n",
      "Segmentation\n",
      "\n",
      "Analysis\n",
      "type\n",
      "\n",
      "Start\n",
      "\n",
      "End\n",
      "\n",
      "Cytoband\n",
      "\n",
      "Length (bps)\n",
      "\n",
      "Copy\n",
      "Number\n",
      "\n",
      "# Markers\n",
      "\n",
      "Genes\n",
      "\n",
      "MM 1\n",
      "MM 2\n",
      "MM 3\n",
      "MM 4\n",
      "MM 5\n",
      "MM 6\n",
      "MM 7\n",
      "MM 8\n",
      "MM 9\n",
      "MM 10\n",
      "MM 11\n",
      "MM 12\n",
      "MM 13\n",
      "MM 14\n",
      "MM 15\n",
      "MM 16\n",
      "MM 17\n",
      "MM 18\n",
      "MM 19\n",
      "MM 20\n",
      "MM 21\n",
      "MM 22\n",
      "MM 23\n",
      "MM 24\n",
      "MM 25\n",
      "MM 26\n",
      "\n",
      "\u000213\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "Del\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "N\n",
      "\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Unpaired\n",
      "Unpaired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Unpaired\n",
      "Unpaired\n",
      "Unpaired\n",
      "Paired\n",
      "Unpaired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "Paired\n",
      "\n",
      "42503650\n",
      "27313597\n",
      "47862787\n",
      "45560358\n",
      "48984705\n",
      "43781771\n",
      "40498617\n",
      "47786017\n",
      "39203461\n",
      "40531599\n",
      "45324420\n",
      "22372135\n",
      "49431816\n",
      "49024071\n",
      "26462920\n",
      "34866084\n",
      "49358246\n",
      "37338941\n",
      "48984705\n",
      "48951859\n",
      "\n",
      "57487878\n",
      "57610331\n",
      "114121253\n",
      "52963785\n",
      "53079298\n",
      "70193498\n",
      "70309383\n",
      "104600044\n",
      "66142848\n",
      "53088969\n",
      "52963785\n",
      "78494745\n",
      "49631113\n",
      "52210918\n",
      "104775995\n",
      "97736842\n",
      "49631113\n",
      "52972100\n",
      "52958855\n",
      "68490076\n",
      "\n",
      "13ql4.11–13q21.1\n",
      "13ql2.2–13q21.2\n",
      "13ql4.2–13q34\n",
      "13ql4.12–13q21.1\n",
      "13ql4.3–13q21.1\n",
      "13ql4.11–13q21.33\n",
      "13ql4.11–13q21.33\n",
      "13ql4.2–13q33.2\n",
      "13ql3.3–13q21.32\n",
      "13ql4.11–13q21.1\n",
      "13ql4.12–13q21.1\n",
      "13ql2.12–13q31.1\n",
      "13ql4.3\n",
      "13ql4.3–13q21.1\n",
      "13ql2.13–13q33.2\n",
      "13ql3.3–13q32.2\n",
      "13ql4.3\n",
      "13ql3.3–13q21.1\n",
      "13ql4.3–13q21.1\n",
      "13ql4.3–13q21.33\n",
      "\n",
      "14984228\n",
      "30296734\n",
      "66258466\n",
      "7403427\n",
      "4094593\n",
      "26411727\n",
      "29810766\n",
      "56814027\n",
      "26939387\n",
      "12557370\n",
      "7639365\n",
      "56122610\n",
      "199297\n",
      "3186847\n",
      "78313075\n",
      "62870758\n",
      "272867\n",
      "15633159\n",
      "3974150\n",
      "19538217\n",
      "\n",
      "0.6930\n",
      "1.0028\n",
      "0.9633\n",
      "0.8485\n",
      "0.7389\n",
      "0.6852\n",
      "0.5797\n",
      "1.2421\n",
      "0.9434\n",
      "0.9634\n",
      "0.9586\n",
      "1.0474\n",
      "1.0060\n",
      "1.2947\n",
      "0.8103\n",
      "1.4238\n",
      "0.5809\n",
      "1.0293\n",
      "1.3794\n",
      "1.5462\n",
      "\n",
      "2789\n",
      "6342\n",
      "13801\n",
      "1488\n",
      "828\n",
      "4764\n",
      "5616\n",
      "11446\n",
      "5040\n",
      "2661\n",
      "1529\n",
      "11979\n",
      "19\n",
      "572\n",
      "16439\n",
      "12446\n",
      "23\n",
      "3373\n",
      "796\n",
      "3374\n",
      "\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "KCNRG, TRIM13\n",
      "ARL11, C13orﬂ, EBPL, KCNRG, KPNA3, RCBTB1, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "C13orﬂ, KCNRG, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "ARL11, C13orﬂ, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, TRIM13\n",
      "ARL11, C13orﬂ, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\n",
      "\n",
      "N indicates normal; Del, deletion; A13, deletion; Amp, ampliﬁcation; ND, not done.\n",
      "\n",
      "S.L. Corthals et al. / Leukemia Research 34 (2010) 677–681\n",
      "\n",
      "Patient\n",
      "\n",
      "\f",
      "S.L. Corthals et al. / Leukemia Research 34 (2010) 677–681\n",
      "\n",
      "679\n",
      "\n",
      "Fig. 1. Copy number alterations of chromosome 13 as determined by paired segmentation analysis. SNP mapping arrays (Illumina Inﬁnium HumanHap550 Genotyping\n",
      "BeadChip) were used to determine the exact deleted regions at chromosome 13 of multiple myeloma (MM) cases. (A) In panel 1; the genome browser illustrates the gene\n",
      "transcripts (NCBI) located on chromosome 13. The results of the paired segmentation analysis of chromosome 13 (20/26 MM patients) are shown in the panel 2. (B) A region\n",
      "on chromosome 13; 13q14.3, including the miR-15a and miR-16 locus, is depicted at larger magniﬁcation. In panel 1 the miR-15a and miR-16 locus (star) is shown within the\n",
      "genome browser. Panel 2 shows the results of the paired segmentation analysis in more detail. A deletion (orange and purple) was detected in 15/20 MM patients. Uniparental\n",
      "disomy (UPD; green) was detected in patient MM 17, in the regions ﬂanking miR-15a and miR-16-1. Copy number (CN) values for SNPs (red dots) are plotted in panel 3. CN\n",
      "values of 2 represent diploid copy number, whereas values of 1 and 0 represent heterozygous and homozygous deletion, respectively. A representative patient (MM 7) shows\n",
      "values of 1 or ≤1, indicating deletion. In panel 4 the B allele frequency is displayed. Each point (green dot) represents the B allele frequency of one SNP. B allele frequencies of 0\n",
      "and 1 represent homozygous signals (AA or BB), while an allele frequency of ∼0.5 represents an equal contribution from both alleles (AB). A representative MM patient sample\n",
      "(MM 7) shows a deletion with LOH (black box in segmentation panel) which produces a characteristic pattern that lacks heterozygote signal. For visualization purposes the\n",
      "order of the samples is MM 26, MM 10, MM 4, MM 1, MM 21, MM 8, MM 14, MM 7. MM 16, MM 15, MM 9, MM 13, MM 2, MM 23, MM 5, MM 24, MM 25, MM 3, and MM 17.\n",
      "genomic blood DNA MM samples). CN, AsCN, LOH and allelic ratios were integrated\n",
      "and visualized in a genomic browser within Partek Genomics Suite.\n",
      "Chromosome analysis was performed using both QFQ- and RBA-banding [17].\n",
      "Fluorescence in situ hybridization (FISH) was performed using standard protocols\n",
      "[18].\n",
      "In this study, the stability of ﬁve candidate reference small nuclear RNAs was\n",
      "examined using the validation program GeNorm [19]. The relative expression levels\n",
      "of miR-15a and miR-16 compared to CD138 sorted plasma cells from normal bone\n",
      "marrow were determined using the 2−\u0002\u0002Ct method [20].\n",
      "The Mann–Whitney U-test was applied to determine differences of the miR-15a\n",
      "or miR-16 gene expression levels.\n",
      "\n",
      "3. Results\n",
      "FISH analysis showed chromosome 13 deletions in 11/26\n",
      "patients (Table 1 and Table S1). Using karyotyping, 1 additional patient had a chromosome 13 deletion, resulting in 12/26\n",
      "patients (46.2%) having a chromosome 13 aberration. CNV analysis using SNP mapping arrays performed on 20/26 paired samples\n",
      "indicated that a minimally deleted region on chromosome 13\n",
      "(49431816–49631113 bp), in which miR-15a and miR-16-1 are\n",
      "\n",
      "\f",
      "680\n",
      "\n",
      "S.L. Corthals et al. / Leukemia Research 34 (2010) 677–681\n",
      "\n",
      "Fig. 2. Copy number of chromosome 13 in correlation with miR-15a gene expression in multiple myeloma (MM) patient samples. A TaqMan miR assay was used to determine\n",
      "the relative miR-15a expression levels in myeloma plasma of 26 MM patients. The normalized relative expression of miR-15a in 26 MM patient samples is displayed on\n",
      "the X-axis. MiR-15a is expressed in all 26 MM samples, although the level of expression varies across the samples. SNP mapping arrays (Illumina Inﬁnium HumanHap550\n",
      "Genotyping BeadChip) were used to determine copy number values of chromosome 13, which are shown on the Y-axis. Copy number values of 2 were detected in 6/26\n",
      "MM patients, indicating diploid copy number. Values less than 2 represent heterozygous and homozygous deletions, which were found in 20/26 MM patients. Patients are\n",
      "coloured by chromosome 13 status as determined by FISH; being either normal (black) or deletion (gray). Segmentation analysis has shown 8 additional patients with a\n",
      "chromosome 13 aberration in the region containing miR-15a and miR-16-1. A linear regression line reveals there is no correlation between the miR-15a expression and the\n",
      "chromosome 13 status as determined by segmentation analysis. The picture insert shows a box plot representation of the normalized relative expression of miR-15a in 26\n",
      "MM patients samples (X-axis), in correlation with the chromosome 13 status determined by segmentation (Y-axis). The box plot shows no signiﬁcant difference in miR-15a\n",
      "gene expression (P = 0.38, Mann–Whitney U-test) between patients with a deletion (gray, n = 20) and normal chromosome 13 status (black, n = 6).\n",
      "\n",
      "located, was observed in 20/26 patients, 15 of which were from\n",
      "the paired analysis (P < 0.00001) (Table 1 and Tables S1 and S2 and\n",
      "Fig. 1, Fig. S1). Segmentation analysis revealed 8 additional patients\n",
      "with a chromosome 13 deletion compared to FISH analysis. AsCN\n",
      "analysis further revealed the absence of homozygous deletions in\n",
      "these patients. No uniparental disomy (UPD; copy number-neutral\n",
      "LOH), was detected in the exact location of miR-15a and miR-161 (Fig. 1 and Fig. S1). A highly correlated cluster of miRs; miR-15b\n",
      "and miR-16-2, is located on chromosome 3. MiR-15a and miR-15b\n",
      "are derived from the same seed sequence, however, they differ in\n",
      "their mature sequence and therefore cannot be distinguished. For\n",
      "this reason, we are not able to separate miR-16-1 and 16-2 expression using the real-time PCR assay, since the mature miR sequence\n",
      "primes the assay. As these highly correlated miRs are located on\n",
      "chromosome 3, we have also examined this region. Analysis of this\n",
      "chromosome was performed using the segmentation algorithm to\n",
      "determine the ampliﬁed and deleted regions (miR-15b and miR16-2 location 161605070–161605307 bp). Deletion of this region\n",
      "on chromosome 3 was found in 2 MM and ampliﬁcation in 6 MM\n",
      "patients (Table S2).\n",
      "Both miR-15a and miR-16 were expressed in all 26 MM samples.\n",
      "The median expression value of miR-15a and miR-16 was 0.84 and\n",
      "1.16, respectively (both values in log2 scale, relative to the geometric mean). No signiﬁcant association was found between the\n",
      "chromosome 13 deletion status and the miR-15a expression levels\n",
      "using FISH (P = 0.38) and CNV (P = 0.25) (Fig. 2). In addition, statistical analysis showed no signiﬁcant association between miR-16\n",
      "expression and chromosome 13 status by FISH (P = 0.40) or CNV\n",
      "\n",
      "(P = 0.27). We also evaluated the expression levels of miR-15a and\n",
      "miR-16 in CD138 sorted plasma cells from normal individuals. MiR16 was expressed at 1.4-fold lower levels in normal plasma cells\n",
      "when compared with myeloma plasma cells.\n",
      "These ﬁndings demonstrate that miR-15a and miR-16 are displayed at a range of expression levels in MM patients which are\n",
      "higher than in normal plasma cell counterparts. The expression of\n",
      "these miRs varies independent of the chromosome 13 status.\n",
      "\n",
      "4. Discussion\n",
      "Deletion of chromosome 13 is detected in 50% of patients by\n",
      "FISH and in 10–20% by karyotyping [5,21–23]. Patients with a whole\n",
      "chromosome 13 deletion detected by cytogenetics, have a worse\n",
      "prognosis than 13q deletions detected by FISH [24–27]. Loss of\n",
      "chromosome 13 occurs, like in MM, in approximately 50% of CLL\n",
      "patients, although CLL patients with 13q deletions have favorable\n",
      "survival [28]. Differences of the length of the involved region may\n",
      "explain these observations [29].\n",
      "Downregulation of miR-15a and miR-16-1 has been reported in\n",
      "cases of MM, CLL and diffuse B Cell Lymphomas [14,15,30]. In this\n",
      "study, the two miRs were expressed in all 26 MM patients examined, even when chromosome 13 was deleted. Our analysis showed\n",
      "an high speciﬁcity for detecting allelic imbalances in heterogeneous\n",
      "samples [31,32]. It is unlikely that chromosome 13 deletions that\n",
      "are not detected by CNV analysis are partial deletions. Compensation by the non-deleted allele could explain the expression level\n",
      "\n",
      "\f",
      "S.L. Corthals et al. / Leukemia Research 34 (2010) 677–681\n",
      "\n",
      "of miR-16 since all chromosome 13 deletions in this study are heterozygous.\n",
      "This study clearly shows the discrepancy between FISH and\n",
      "CNV analysis and the efﬁcacy of SNP mapping arrays in detecting\n",
      "chromosomal aberration. It is important that a direct comparison\n",
      "between tumor DNA and matched germline DNA was performed\n",
      "[33]. Because paired CN analysis was carried out for 20/26 MM\n",
      "samples, this makes the data highly reliable. In conclusion, high\n",
      "resolution, genome-wide SNP mapping arrays may provide an\n",
      "excellent tool to identify partial chromosomal aberrations and\n",
      "genes.\n",
      "Conﬂict of interest statement\n",
      "There are no conﬂict of interests in relation to this work.\n",
      "Appendix A. Supplementary data\n",
      "Supplementary data associated with this article can be found, in\n",
      "the online version, at doi:10.1016/j.leukres.2009.10.026.\n",
      "References\n",
      "[1] Kyle RA, Rajkumar SV. Multiple myeloma. N Engl J Med 2004;351(18):1860–73.\n",
      "[2] Avet-Louseau H, Daviet A, Sauner S, Bataille R. Chromosome 13 abnormalities in multiple myeloma are mostly monosomy 13. Br J Haematol\n",
      "2000;111(4):1116–7.\n",
      "[3] Shaughnessy J, Tian E, Sawyer J, Bumm K, Landes R, Badros A, et al. High incidence of chromosome 13 deletion in multiple myeloma detected by multiprobe\n",
      "interphase FISH. Blood 2000;96(4):1505–11.\n",
      "[4] Debes-Marun CS, Dewald GW, Bryant S, Picken E, Santana-Davila R, GonzalezPaz N, et al. Chromosome abnormalities clustering and its implications for\n",
      "pathogenesis and prognosis in myeloma. Leukemia 2003;17(2):427–36.\n",
      "[5] Facon T, Avet-Loiseau H, Guillerm G, Moreau P, Genevieve F, Zandecki M,\n",
      "et al. Chromosome 13 abnormalities identiﬁed by FISH analysis and serum\n",
      "beta2-microglobulin produce a powerful myeloma staging system for patients\n",
      "receiving high-dose therapy. Blood 2001;97(6):1566–71.\n",
      "[6] Fonseca R, Harrington D, Oken MM, Dewald GW, Bailey RJ, Van Wier SA, et\n",
      "al. Biological and prognostic signiﬁcance of interphase ﬂuorescence in situ\n",
      "hybridization detection of chromosome 13 abnormalities (delta13) in multiple myeloma: an eastern cooperative oncology group study. Cancer Res\n",
      "2002;62(3):715–20.\n",
      "[7] Seong C, Delasalle K, Hayes K, Weber D, Dimopoulos M, Swantkowski J,\n",
      "et al. Prognostic value of cytogenetics in multiple myeloma. Br J Haematol\n",
      "1998;101(1):189–94.\n",
      "[8] Shaughnessy J, Barlogie B. Chromosome 13 deletion in myeloma. Curr Top\n",
      "Microbiol Immunol 1999;246:199–203.\n",
      "[9] Tricot G, Barlogie B, Jagannath S, Bracy D, Mattox S, Vesole DH, et al. Poor prognosis in multiple myeloma is associated only with partial or complete deletions\n",
      "of chromosome 13 or abnormalities involving 11q and not with other karyotype\n",
      "abnormalities. Blood 1995;86(11):4250–6.\n",
      "[10] Zojer N, Konigsberg R, Ackermann J, Fritz E, Dallinger S, Kromer E, et al. Deletion of 13q14 remains an independent adverse prognostic variable in multiple\n",
      "myeloma despite its frequent detection by interphase ﬂuorescence in situ\n",
      "hybridization. Blood 2000;95(6):1925–30.\n",
      "[11] Grifﬁths-Jones S, Grocock RJ, van Dongen S, Bateman A, Enright AJ. miRBase: microRNA sequences, targets and gene nomenclature. Nucleic Acids Res\n",
      "2006;34:D140–4 [Database issue].\n",
      "[12] Bartel DP. MicroRNAs: genomics, biogenesis, mechanism, and function. Cell\n",
      "2004;116(2):281–97.\n",
      "\n",
      "681\n",
      "\n",
      "[13] Pichiorri F, Suh SS, Ladetto M, Kuehl M, Palumbo T, Drandi D, et al. MicroRNAs\n",
      "regulate critical genes associated with multiple myeloma pathogenesis. Proc\n",
      "Natl Acad Sci USA 2008;105(35):12885–90.\n",
      "[14] Roccaro AM, Sacco A, Thompson B, Leleu X, Azab AK, Azab F, et al. microRNAs\n",
      "15a and 16 regulate tumor proliferation in multiple myeloma. Blood 2009.\n",
      "[15] Calin GA, Dumitru CD, Shimizu M, Bichi R, Zupo S, Noch E, et al. Frequent deletions and down-regulation of micro-RNA genes miR15 and miR16 at 13q14 in\n",
      "chronic lymphocytic leukemia. Proc Natl Acad Sci USA 2002;99(24):15524–9.\n",
      "[16] Chen C, Ridzon DA, Broomer AJ, Zhou Z, Lee DH, Nguyen JT, et al. Realtime quantiﬁcation of microRNAs by stem-loop RT-PCR. Nucleic Acids Res\n",
      "2005;33(20):e179.\n",
      "[17] Schaffer LG, Tommerup N, ISCN. An International System for Human Cytogenetic Nomenclature. S. Karger; 2005.\n",
      "[18] van Zutven LJ, Velthuizen SC, Wolvers-Tettero IL, van Dongen JJ, Poulsen TS,\n",
      "MacLeod RA, et al. Two dual-color split signal ﬂuorescence in situ hybridization\n",
      "assays to detect t(5;14) involving HOX11L2 or CSX in T-cell acute lymphoblastic\n",
      "leukemia. Haematologica 2004;89(6):671–8.\n",
      "[19] Vandesompele J, De Preter K, Pattyn F, Poppe B, Van Roy N, De Paepe A,\n",
      "et al. Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome Biol 2002;3(7)\n",
      "[RESEARCH0034].\n",
      "[20] Livak KJ, Schmittgen TD. Analysis of relative gene expression data using\n",
      "real-time quantitative PCR and the 2(-Delta Delta C(T)) method. Methods\n",
      "2001;25(4):402–8.\n",
      "[21] Chng WJ, Santana-Davila R, Van Wier SA, Ahmann GJ, Jalal SM, Bergsagel PL,\n",
      "et al. Prognostic factors for hyperdiploid-myeloma: effects of chromosome 13\n",
      "deletions and IgH translocations. Leukemia 2006;20(5):807–13.\n",
      "[22] Chang H, Bouman D, Boerkoel CF, Stewart AK, Squire JA. Frequent monoallelic\n",
      "loss of D13S319 in multiple myeloma patients shown by interphase ﬂuorescence in situ hybridization. Leukemia 1999;13(1):105–9.\n",
      "[23] Fassas AB, Spencer T, Sawyer J, Zangari M, Lee CK, Anaissie E, et al. Both\n",
      "hypodiploidy and deletion of chromosome 13 independently confer poor prognosis in multiple myeloma. Br J Haematol 2002;118(4):1041–7.\n",
      "[24] Kaufmann H, Kromer E, Nosslinger T, Weltermann A, Ackermann J, Reisner\n",
      "R, et al. Both chromosome 13 abnormalities by metaphase cytogenetics and\n",
      "deletion of 13q by interphase FISH only are prognostically relevant in multiple\n",
      "myeloma. Eur J Haematol 2003;71(3):179–83.\n",
      "[25] Shaughnessy Jr J, Tian E, Sawyer J, McCoy J, Tricot G, Jacobson J, et al. Prognostic\n",
      "impact of cytogenetic and interphase ﬂuorescence in situ hybridizationdeﬁned chromosome 13 deletion in multiple myeloma: early results of total\n",
      "therapy II. Br J Haematol 2003;120(1):44–52.\n",
      "[26] Dewald GW, Therneau T, Larson D, Lee YK, Fink S, Smoley S, et al. Relationship\n",
      "of patient survival and chromosome anomalies detected in metaphase and/or\n",
      "interphase cells at diagnosis of myeloma. Blood 2005;106(10):3553–8.\n",
      "[27] Chiecchio L, Protheroe RK, Ibrahim AH, Cheung KL, Rudduck C, Dagrada GP,\n",
      "et al. Deletion of chromosome 13 detected by conventional cytogenetics is a\n",
      "critical prognostic factor in myeloma. Leukemia 2006;20(9):1610–7.\n",
      "[28] Dohner H, Stilgenbauer S, Benner A, Leupolt E, Krober A, Bullinger L, et al.\n",
      "Genomic aberrations and survival in chronic lymphocytic leukemia. N Engl J\n",
      "Med 2000;343(26):1910–6.\n",
      "[29] Rowntree C, Duke V, Panayiotidis P, Kotsi P, Palmisano GL, Hoffbrand AV, et\n",
      "al. Deletion analysis of chromosome 13q14.3 and characterisation of an alternative splice form of LEU1 in B cell chronic lymphocytic leukemia. Leukemia\n",
      "2002;16(7):1267–75.\n",
      "[30] Eis PS, Tam W, Sun L, Chadburn A, Li Z, Gomez MF, et al. Accumulation of\n",
      "miR-155 and BIC RNA in human B cell lymphomas. Proc Natl Acad Sci USA\n",
      "2005;102(10):3627–32.\n",
      "[31] Staaf J, Lindgren D, Vallon-Christersson J, Isaksson A, Goransson H, Juliusson\n",
      "G, et al. Segmentation-based detection of allelic imbalance and loss-ofheterozygosity in cancer cells using whole genome SNP arrays. Genome Biol\n",
      "2008;9(9):R136.\n",
      "[32] Lai WR, Johnson MD, Kucherlapati R, Park PJ. Comparative analysis of\n",
      "algorithms for identifying ampliﬁcations and deletions in array CGH data.\n",
      "Bioinformatics 2005;21(19):3763–70.\n",
      "[33] Feng S, Liang Q, Kinser RD, Newland K, Guilbaud R. Testing equivalence between\n",
      "two laboratories or two methods using paired-sample analysis and interval\n",
      "hypothesis testing. Anal Bioanal Chem 2006;385(5):975–81.\n",
      "\n",
      "\f",
      "\n"
     ]
    }
   ],
   "source": [
    "text = textract.process(\"./pdfs/paper.pdf\")\n",
    "print(text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Leukemia Research 34 (2010) 677\\xe2\\x80\\x93681\\n\\nContents lists available at ScienceDirect\\n\\nLeukemia Research\\njournal homepage: www.elsevier.com/locate/leukres\\n\\nBrief communication\\n\\nMicro-RNA-15a and micro-RNA-16 expression and chromosome 13 deletions in\\nmultiple myeloma\\nSophie L. Corthals a , Mojca Jongen-Lavrencic a , Yvonne de Knegt a , Justine K. Peeters a ,\\nH. Berna Beverloo b , Henk M. Lokhorst c , Pieter Sonneveld a,\\xe2\\x88\\x97\\na\\nb\\nc\\n\\nDepartment of Hematology, Erasmus Medical Centre Rotterdam, Rotterdam, The Netherlands\\nDepartment of Clinical Genetics, Erasmus Medical Centre Rotterdam, Rotterdam, The Netherlands\\nDepartment of Hematology, University Medical Center Utrecht, Utrecht, The Netherlands\\n\\na r t i c l e\\n\\ni n f o\\n\\nArticle history:\\nReceived 15 October 2009\\nReceived in revised form 28 October 2009\\nAccepted 28 October 2009\\nAvailable online 23 December 2009\\nKeywords:\\nMultiple myeloma\\nMicro-RNA\\nChromosome 13 deletion\\n\\na b s t r a c t\\nWe have used copy number variation (CNV) analysis with SNP mapping arrays for miRNA-15a and\\nmiRNA-16-1 expression analysis in patients with multiple myeloma (MM) with or without deletion of\\nchromosome 13q14. MiRNA-15a and miRNA-16 display a range of expression patterns in MM patients,\\nindependent of the chromosome 13 status. These \\xef\\xac\\x81ndings suggest that genes other than miR-15a and\\nmiR-16 may explain the prognostic signi\\xef\\xac\\x81cance of 13q14 deletions.\\n\\xc2\\xa9 2009 Elsevier Ltd. All rights reserved.\\n\\n1. Introduction\\nMultiple myeloma (MM) is a dismal disease with a median survival of 3\\xe2\\x80\\x935 years [1].\\nMM is characterized by genetic instability and chromosomal\\ntranslocations early in disease progression play an important role\\nin the pathogenesis of MM. Chromosome 13 aberrations occur in\\n50% and the most commonly deleted marker in the 13q14 region is\\nD13S319, located distal to the retinoblastoma-1 (RB1) gene [2,3]. It\\nis now common practice to determine the presence of chromosome\\n13 deletions by \\xef\\xac\\x82uorescence in situ hybridization (FISH) [4\\xe2\\x80\\x9310].\\nThe adverse prognostic role of these deletions has been attributed\\nto the presence of MM tumor suppressor gene(s) on chromosome\\n13q which as yet have not been identi\\xef\\xac\\x81ed.\\nMicro-RNAs (miRs) are a class of small non-coding single\\nstranded RNAs of approximately 22 nucleotides in length [11]. MiRs\\nnegatively regulate gene expression by binding to partially complementary sites in messenger RNAs (mRNAs) [12]. miRs may play\\na role in the pathogenesis of MM [13,14]. In chronic lymphocytic\\nleukemia (CLL), miR-15a and miR-16-1 are frequently downregu-\\n\\n\\xe2\\x88\\x97 Corresponding author at: Department of Hematology, Erasmus Medical Centre\\nRotterdam, Room No. L439, PO Box 2040, 3000 CA Rotterdam, The Netherlands.\\nTel.: +31 10 7033123; fax: +31 10 7035814.\\nE-mail address: p.sonneveld@erasmusmc.nl (P. Sonneveld).\\n0145-2126/$ \\xe2\\x80\\x93 see front matter \\xc2\\xa9 2009 Elsevier Ltd. All rights reserved.\\ndoi:10.1016/j.leukres.2009.10.026\\n\\nlated [15]. We evaluated the expression of miR-15a and miR-16-1\\nin MM.\\n2. Materials and methods\\nWe obtained bone marrow samples from newly diagnosed patients with MM.\\nMyeloma plasma cells were puri\\xef\\xac\\x81ed >80% using CD138 magnetic microbeads\\n(MACS system, Miltenyi Biotec, Bergisch Gladbach, Germany). Small RNA transcripts less than 200 nucleotides in length including miRs, were isolated from\\nCD138 magnetic cell selected (MACS) MM plasma cells using miRVanaTM miRNA\\nIsolation Kit (Ambion, Austin, TX). RNA levels and quality were assessed using\\nthe Agilent 2100 Bioanalyzer (Agilent Technologies). Mature miR-15a and miR16 expression levels in MM plasma cells were determined by real-time PCR\\nusing TaqMan miR assay (Applied Biosystems, P/N: 4373123 and 4373121)\\n[16]. All experiments were performed in duplicate and miRs were considered\\nas present when CT -values were lower than 35. Genomic DNA was isolated\\nfrom peripheral blood by salting out precipitation. Tumor DNA was extracted\\nfrom CD138 MACS MM plasma cells using Qiagen RNAeasy kit (Qiagen, Valencia, CA). Genome-wide SNP genotyping was performed using the Illumina\\nIn\\xef\\xac\\x81nium HumanHap550 Genotyping BeadChip, containing over 550,000 unique\\ntag SNP markers, according to manufacturer\\xe2\\x80\\x99s instructions (Illumina, San Diego,\\nCA).\\nGenotypes for all arrays were calculated using BeadStudio\\xe2\\x80\\x99s genotyping module (v2.0, Illumina). Data was imported into Partek Genomics Suite 6.4 software\\n(Partek Inc., St Louis, MO) for further analysis; allele intensities were calculated\\nfor 25 genomic DNA MM samples and 26 tumor DNA MM samples, of which 20\\nhad paired normal samples. For these 20 samples, paired analysis was performed.\\nAllele speci\\xef\\xac\\x81c copy number variation (AsCN) was calculated by estimating the number of copies for each allele, rather than total copies of each chromosome. Loss of\\nheterozygosity (LOH) was estimated from the imported sample genotypes by a Hidden Markov Model, using the same baseline as in the segmentation algorithm (25\\n\\n\\x0c678\\n\\nTable 1\\nChromosome 13 aberrations of the region \\xef\\xac\\x82anking miR-15a and miR-16-1 identi\\xef\\xac\\x81ed by FISH and segmentation analysis using SNP mapping arrays.\\nFISH\\n\\nSegmentation\\n\\nAnalysis\\ntype\\n\\nStart\\n\\nEnd\\n\\nCytoband\\n\\nLength (bps)\\n\\nCopy\\nNumber\\n\\n# Markers\\n\\nGenes\\n\\nMM 1\\nMM 2\\nMM 3\\nMM 4\\nMM 5\\nMM 6\\nMM 7\\nMM 8\\nMM 9\\nMM 10\\nMM 11\\nMM 12\\nMM 13\\nMM 14\\nMM 15\\nMM 16\\nMM 17\\nMM 18\\nMM 19\\nMM 20\\nMM 21\\nMM 22\\nMM 23\\nMM 24\\nMM 25\\nMM 26\\n\\n\\x0213\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nN\\nN\\nN\\nN\\nN\\nN\\nN\\nN\\nN\\nN\\nN\\nN\\nN\\nN\\n\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nDel\\nN\\nN\\nN\\nN\\nN\\nN\\n\\nPaired\\nPaired\\nPaired\\nPaired\\nPaired\\nPaired\\nPaired\\nPaired\\nPaired\\nPaired\\nUnpaired\\nUnpaired\\nPaired\\nPaired\\nPaired\\nPaired\\nPaired\\nUnpaired\\nUnpaired\\nUnpaired\\nPaired\\nUnpaired\\nPaired\\nPaired\\nPaired\\nPaired\\n\\n42503650\\n27313597\\n47862787\\n45560358\\n48984705\\n43781771\\n40498617\\n47786017\\n39203461\\n40531599\\n45324420\\n22372135\\n49431816\\n49024071\\n26462920\\n34866084\\n49358246\\n37338941\\n48984705\\n48951859\\n\\n57487878\\n57610331\\n114121253\\n52963785\\n53079298\\n70193498\\n70309383\\n104600044\\n66142848\\n53088969\\n52963785\\n78494745\\n49631113\\n52210918\\n104775995\\n97736842\\n49631113\\n52972100\\n52958855\\n68490076\\n\\n13ql4.11\\xe2\\x80\\x9313q21.1\\n13ql2.2\\xe2\\x80\\x9313q21.2\\n13ql4.2\\xe2\\x80\\x9313q34\\n13ql4.12\\xe2\\x80\\x9313q21.1\\n13ql4.3\\xe2\\x80\\x9313q21.1\\n13ql4.11\\xe2\\x80\\x9313q21.33\\n13ql4.11\\xe2\\x80\\x9313q21.33\\n13ql4.2\\xe2\\x80\\x9313q33.2\\n13ql3.3\\xe2\\x80\\x9313q21.32\\n13ql4.11\\xe2\\x80\\x9313q21.1\\n13ql4.12\\xe2\\x80\\x9313q21.1\\n13ql2.12\\xe2\\x80\\x9313q31.1\\n13ql4.3\\n13ql4.3\\xe2\\x80\\x9313q21.1\\n13ql2.13\\xe2\\x80\\x9313q33.2\\n13ql3.3\\xe2\\x80\\x9313q32.2\\n13ql4.3\\n13ql3.3\\xe2\\x80\\x9313q21.1\\n13ql4.3\\xe2\\x80\\x9313q21.1\\n13ql4.3\\xe2\\x80\\x9313q21.33\\n\\n14984228\\n30296734\\n66258466\\n7403427\\n4094593\\n26411727\\n29810766\\n56814027\\n26939387\\n12557370\\n7639365\\n56122610\\n199297\\n3186847\\n78313075\\n62870758\\n272867\\n15633159\\n3974150\\n19538217\\n\\n0.6930\\n1.0028\\n0.9633\\n0.8485\\n0.7389\\n0.6852\\n0.5797\\n1.2421\\n0.9434\\n0.9634\\n0.9586\\n1.0474\\n1.0060\\n1.2947\\n0.8103\\n1.4238\\n0.5809\\n1.0293\\n1.3794\\n1.5462\\n\\n2789\\n6342\\n13801\\n1488\\n828\\n4764\\n5616\\n11446\\n5040\\n2661\\n1529\\n11979\\n19\\n572\\n16439\\n12446\\n23\\n3373\\n796\\n3374\\n\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nKCNRG, TRIM13\\nARL11, C13or\\xef\\xac\\x82, EBPL, KCNRG, KPNA3, RCBTB1, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nC13or\\xef\\xac\\x82, KCNRG, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\nARL11, C13or\\xef\\xac\\x82, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, TRIM13\\nARL11, C13or\\xef\\xac\\x82, DLEU1, EBPL, KCNRG, KPNA3, PHF11, RCBTB1, SETDB2, TRIM13\\n\\nN indicates normal; Del, deletion; A13, deletion; Amp, ampli\\xef\\xac\\x81cation; ND, not done.\\n\\nS.L. Corthals et al. / Leukemia Research 34 (2010) 677\\xe2\\x80\\x93681\\n\\nPatient\\n\\n\\x0cS.L. Corthals et al. / Leukemia Research 34 (2010) 677\\xe2\\x80\\x93681\\n\\n679\\n\\nFig. 1. Copy number alterations of chromosome 13 as determined by paired segmentation analysis. SNP mapping arrays (Illumina In\\xef\\xac\\x81nium HumanHap550 Genotyping\\nBeadChip) were used to determine the exact deleted regions at chromosome 13 of multiple myeloma (MM) cases. (A) In panel 1; the genome browser illustrates the gene\\ntranscripts (NCBI) located on chromosome 13. The results of the paired segmentation analysis of chromosome 13 (20/26 MM patients) are shown in the panel 2. (B) A region\\non chromosome 13; 13q14.3, including the miR-15a and miR-16 locus, is depicted at larger magni\\xef\\xac\\x81cation. In panel 1 the miR-15a and miR-16 locus (star) is shown within the\\ngenome browser. Panel 2 shows the results of the paired segmentation analysis in more detail. A deletion (orange and purple) was detected in 15/20 MM patients. Uniparental\\ndisomy (UPD; green) was detected in patient MM 17, in the regions \\xef\\xac\\x82anking miR-15a and miR-16-1. Copy number (CN) values for SNPs (red dots) are plotted in panel 3. CN\\nvalues of 2 represent diploid copy number, whereas values of 1 and 0 represent heterozygous and homozygous deletion, respectively. A representative patient (MM 7) shows\\nvalues of 1 or \\xe2\\x89\\xa41, indicating deletion. In panel 4 the B allele frequency is displayed. Each point (green dot) represents the B allele frequency of one SNP. B allele frequencies of 0\\nand 1 represent homozygous signals (AA or BB), while an allele frequency of \\xe2\\x88\\xbc0.5 represents an equal contribution from both alleles (AB). A representative MM patient sample\\n(MM 7) shows a deletion with LOH (black box in segmentation panel) which produces a characteristic pattern that lacks heterozygote signal. For visualization purposes the\\norder of the samples is MM 26, MM 10, MM 4, MM 1, MM 21, MM 8, MM 14, MM 7. MM 16, MM 15, MM 9, MM 13, MM 2, MM 23, MM 5, MM 24, MM 25, MM 3, and MM 17.\\ngenomic blood DNA MM samples). CN, AsCN, LOH and allelic ratios were integrated\\nand visualized in a genomic browser within Partek Genomics Suite.\\nChromosome analysis was performed using both QFQ- and RBA-banding [17].\\nFluorescence in situ hybridization (FISH) was performed using standard protocols\\n[18].\\nIn this study, the stability of \\xef\\xac\\x81ve candidate reference small nuclear RNAs was\\nexamined using the validation program GeNorm [19]. The relative expression levels\\nof miR-15a and miR-16 compared to CD138 sorted plasma cells from normal bone\\nmarrow were determined using the 2\\xe2\\x88\\x92\\x02\\x02Ct method [20].\\nThe Mann\\xe2\\x80\\x93Whitney U-test was applied to determine differences of the miR-15a\\nor miR-16 gene expression levels.\\n\\n3. Results\\nFISH analysis showed chromosome 13 deletions in 11/26\\npatients (Table 1 and Table S1). Using karyotyping, 1 additional patient had a chromosome 13 deletion, resulting in 12/26\\npatients (46.2%) having a chromosome 13 aberration. CNV analysis using SNP mapping arrays performed on 20/26 paired samples\\nindicated that a minimally deleted region on chromosome 13\\n(49431816\\xe2\\x80\\x9349631113 bp), in which miR-15a and miR-16-1 are\\n\\n\\x0c680\\n\\nS.L. Corthals et al. / Leukemia Research 34 (2010) 677\\xe2\\x80\\x93681\\n\\nFig. 2. Copy number of chromosome 13 in correlation with miR-15a gene expression in multiple myeloma (MM) patient samples. A TaqMan miR assay was used to determine\\nthe relative miR-15a expression levels in myeloma plasma of 26 MM patients. The normalized relative expression of miR-15a in 26 MM patient samples is displayed on\\nthe X-axis. MiR-15a is expressed in all 26 MM samples, although the level of expression varies across the samples. SNP mapping arrays (Illumina In\\xef\\xac\\x81nium HumanHap550\\nGenotyping BeadChip) were used to determine copy number values of chromosome 13, which are shown on the Y-axis. Copy number values of 2 were detected in 6/26\\nMM patients, indicating diploid copy number. Values less than 2 represent heterozygous and homozygous deletions, which were found in 20/26 MM patients. Patients are\\ncoloured by chromosome 13 status as determined by FISH; being either normal (black) or deletion (gray). Segmentation analysis has shown 8 additional patients with a\\nchromosome 13 aberration in the region containing miR-15a and miR-16-1. A linear regression line reveals there is no correlation between the miR-15a expression and the\\nchromosome 13 status as determined by segmentation analysis. The picture insert shows a box plot representation of the normalized relative expression of miR-15a in 26\\nMM patients samples (X-axis), in correlation with the chromosome 13 status determined by segmentation (Y-axis). The box plot shows no signi\\xef\\xac\\x81cant difference in miR-15a\\ngene expression (P = 0.38, Mann\\xe2\\x80\\x93Whitney U-test) between patients with a deletion (gray, n = 20) and normal chromosome 13 status (black, n = 6).\\n\\nlocated, was observed in 20/26 patients, 15 of which were from\\nthe paired analysis (P < 0.00001) (Table 1 and Tables S1 and S2 and\\nFig. 1, Fig. S1). Segmentation analysis revealed 8 additional patients\\nwith a chromosome 13 deletion compared to FISH analysis. AsCN\\nanalysis further revealed the absence of homozygous deletions in\\nthese patients. No uniparental disomy (UPD; copy number-neutral\\nLOH), was detected in the exact location of miR-15a and miR-161 (Fig. 1 and Fig. S1). A highly correlated cluster of miRs; miR-15b\\nand miR-16-2, is located on chromosome 3. MiR-15a and miR-15b\\nare derived from the same seed sequence, however, they differ in\\ntheir mature sequence and therefore cannot be distinguished. For\\nthis reason, we are not able to separate miR-16-1 and 16-2 expression using the real-time PCR assay, since the mature miR sequence\\nprimes the assay. As these highly correlated miRs are located on\\nchromosome 3, we have also examined this region. Analysis of this\\nchromosome was performed using the segmentation algorithm to\\ndetermine the ampli\\xef\\xac\\x81ed and deleted regions (miR-15b and miR16-2 location 161605070\\xe2\\x80\\x93161605307 bp). Deletion of this region\\non chromosome 3 was found in 2 MM and ampli\\xef\\xac\\x81cation in 6 MM\\npatients (Table S2).\\nBoth miR-15a and miR-16 were expressed in all 26 MM samples.\\nThe median expression value of miR-15a and miR-16 was 0.84 and\\n1.16, respectively (both values in log2 scale, relative to the geometric mean). No signi\\xef\\xac\\x81cant association was found between the\\nchromosome 13 deletion status and the miR-15a expression levels\\nusing FISH (P = 0.38) and CNV (P = 0.25) (Fig. 2). In addition, statistical analysis showed no signi\\xef\\xac\\x81cant association between miR-16\\nexpression and chromosome 13 status by FISH (P = 0.40) or CNV\\n\\n(P = 0.27). We also evaluated the expression levels of miR-15a and\\nmiR-16 in CD138 sorted plasma cells from normal individuals. MiR16 was expressed at 1.4-fold lower levels in normal plasma cells\\nwhen compared with myeloma plasma cells.\\nThese \\xef\\xac\\x81ndings demonstrate that miR-15a and miR-16 are displayed at a range of expression levels in MM patients which are\\nhigher than in normal plasma cell counterparts. The expression of\\nthese miRs varies independent of the chromosome 13 status.\\n\\n4. Discussion\\nDeletion of chromosome 13 is detected in 50% of patients by\\nFISH and in 10\\xe2\\x80\\x9320% by karyotyping [5,21\\xe2\\x80\\x9323]. Patients with a whole\\nchromosome 13 deletion detected by cytogenetics, have a worse\\nprognosis than 13q deletions detected by FISH [24\\xe2\\x80\\x9327]. Loss of\\nchromosome 13 occurs, like in MM, in approximately 50% of CLL\\npatients, although CLL patients with 13q deletions have favorable\\nsurvival [28]. Differences of the length of the involved region may\\nexplain these observations [29].\\nDownregulation of miR-15a and miR-16-1 has been reported in\\ncases of MM, CLL and diffuse B Cell Lymphomas [14,15,30]. In this\\nstudy, the two miRs were expressed in all 26 MM patients examined, even when chromosome 13 was deleted. Our analysis showed\\nan high speci\\xef\\xac\\x81city for detecting allelic imbalances in heterogeneous\\nsamples [31,32]. It is unlikely that chromosome 13 deletions that\\nare not detected by CNV analysis are partial deletions. Compensation by the non-deleted allele could explain the expression level\\n\\n\\x0cS.L. Corthals et al. / Leukemia Research 34 (2010) 677\\xe2\\x80\\x93681\\n\\nof miR-16 since all chromosome 13 deletions in this study are heterozygous.\\nThis study clearly shows the discrepancy between FISH and\\nCNV analysis and the ef\\xef\\xac\\x81cacy of SNP mapping arrays in detecting\\nchromosomal aberration. It is important that a direct comparison\\nbetween tumor DNA and matched germline DNA was performed\\n[33]. Because paired CN analysis was carried out for 20/26 MM\\nsamples, this makes the data highly reliable. In conclusion, high\\nresolution, genome-wide SNP mapping arrays may provide an\\nexcellent tool to identify partial chromosomal aberrations and\\ngenes.\\nCon\\xef\\xac\\x82ict of interest statement\\nThere are no con\\xef\\xac\\x82ict of interests in relation to this work.\\nAppendix A. Supplementary data\\nSupplementary data associated with this article can be found, in\\nthe online version, at doi:10.1016/j.leukres.2009.10.026.\\nReferences\\n[1] Kyle RA, Rajkumar SV. Multiple myeloma. N Engl J Med 2004;351(18):1860\\xe2\\x80\\x9373.\\n[2] Avet-Louseau H, Daviet A, Sauner S, Bataille R. Chromosome 13 abnormalities in multiple myeloma are mostly monosomy 13. Br J Haematol\\n2000;111(4):1116\\xe2\\x80\\x937.\\n[3] Shaughnessy J, Tian E, Sawyer J, Bumm K, Landes R, Badros A, et al. High incidence of chromosome 13 deletion in multiple myeloma detected by multiprobe\\ninterphase FISH. Blood 2000;96(4):1505\\xe2\\x80\\x9311.\\n[4] Debes-Marun CS, Dewald GW, Bryant S, Picken E, Santana-Davila R, GonzalezPaz N, et al. Chromosome abnormalities clustering and its implications for\\npathogenesis and prognosis in myeloma. Leukemia 2003;17(2):427\\xe2\\x80\\x9336.\\n[5] Facon T, Avet-Loiseau H, Guillerm G, Moreau P, Genevieve F, Zandecki M,\\net al. Chromosome 13 abnormalities identi\\xef\\xac\\x81ed by FISH analysis and serum\\nbeta2-microglobulin produce a powerful myeloma staging system for patients\\nreceiving high-dose therapy. Blood 2001;97(6):1566\\xe2\\x80\\x9371.\\n[6] Fonseca R, Harrington D, Oken MM, Dewald GW, Bailey RJ, Van Wier SA, et\\nal. Biological and prognostic signi\\xef\\xac\\x81cance of interphase \\xef\\xac\\x82uorescence in situ\\nhybridization detection of chromosome 13 abnormalities (delta13) in multiple myeloma: an eastern cooperative oncology group study. Cancer Res\\n2002;62(3):715\\xe2\\x80\\x9320.\\n[7] Seong C, Delasalle K, Hayes K, Weber D, Dimopoulos M, Swantkowski J,\\net al. Prognostic value of cytogenetics in multiple myeloma. Br J Haematol\\n1998;101(1):189\\xe2\\x80\\x9394.\\n[8] Shaughnessy J, Barlogie B. Chromosome 13 deletion in myeloma. Curr Top\\nMicrobiol Immunol 1999;246:199\\xe2\\x80\\x93203.\\n[9] Tricot G, Barlogie B, Jagannath S, Bracy D, Mattox S, Vesole DH, et al. Poor prognosis in multiple myeloma is associated only with partial or complete deletions\\nof chromosome 13 or abnormalities involving 11q and not with other karyotype\\nabnormalities. Blood 1995;86(11):4250\\xe2\\x80\\x936.\\n[10] Zojer N, Konigsberg R, Ackermann J, Fritz E, Dallinger S, Kromer E, et al. Deletion of 13q14 remains an independent adverse prognostic variable in multiple\\nmyeloma despite its frequent detection by interphase \\xef\\xac\\x82uorescence in situ\\nhybridization. Blood 2000;95(6):1925\\xe2\\x80\\x9330.\\n[11] Grif\\xef\\xac\\x81ths-Jones S, Grocock RJ, van Dongen S, Bateman A, Enright AJ. miRBase: microRNA sequences, targets and gene nomenclature. Nucleic Acids Res\\n2006;34:D140\\xe2\\x80\\x934 [Database issue].\\n[12] Bartel DP. MicroRNAs: genomics, biogenesis, mechanism, and function. Cell\\n2004;116(2):281\\xe2\\x80\\x9397.\\n\\n681\\n\\n[13] Pichiorri F, Suh SS, Ladetto M, Kuehl M, Palumbo T, Drandi D, et al. MicroRNAs\\nregulate critical genes associated with multiple myeloma pathogenesis. Proc\\nNatl Acad Sci USA 2008;105(35):12885\\xe2\\x80\\x9390.\\n[14] Roccaro AM, Sacco A, Thompson B, Leleu X, Azab AK, Azab F, et al. microRNAs\\n15a and 16 regulate tumor proliferation in multiple myeloma. Blood 2009.\\n[15] Calin GA, Dumitru CD, Shimizu M, Bichi R, Zupo S, Noch E, et al. Frequent deletions and down-regulation of micro-RNA genes miR15 and miR16 at 13q14 in\\nchronic lymphocytic leukemia. Proc Natl Acad Sci USA 2002;99(24):15524\\xe2\\x80\\x939.\\n[16] Chen C, Ridzon DA, Broomer AJ, Zhou Z, Lee DH, Nguyen JT, et al. Realtime quanti\\xef\\xac\\x81cation of microRNAs by stem-loop RT-PCR. Nucleic Acids Res\\n2005;33(20):e179.\\n[17] Schaffer LG, Tommerup N, ISCN. An International System for Human Cytogenetic Nomenclature. S. Karger; 2005.\\n[18] van Zutven LJ, Velthuizen SC, Wolvers-Tettero IL, van Dongen JJ, Poulsen TS,\\nMacLeod RA, et al. Two dual-color split signal \\xef\\xac\\x82uorescence in situ hybridization\\nassays to detect t(5;14) involving HOX11L2 or CSX in T-cell acute lymphoblastic\\nleukemia. Haematologica 2004;89(6):671\\xe2\\x80\\x938.\\n[19] Vandesompele J, De Preter K, Pattyn F, Poppe B, Van Roy N, De Paepe A,\\net al. Accurate normalization of real-time quantitative RT-PCR data by geometric averaging of multiple internal control genes. Genome Biol 2002;3(7)\\n[RESEARCH0034].\\n[20] Livak KJ, Schmittgen TD. Analysis of relative gene expression data using\\nreal-time quantitative PCR and the 2(-Delta Delta C(T)) method. Methods\\n2001;25(4):402\\xe2\\x80\\x938.\\n[21] Chng WJ, Santana-Davila R, Van Wier SA, Ahmann GJ, Jalal SM, Bergsagel PL,\\net al. Prognostic factors for hyperdiploid-myeloma: effects of chromosome 13\\ndeletions and IgH translocations. Leukemia 2006;20(5):807\\xe2\\x80\\x9313.\\n[22] Chang H, Bouman D, Boerkoel CF, Stewart AK, Squire JA. Frequent monoallelic\\nloss of D13S319 in multiple myeloma patients shown by interphase \\xef\\xac\\x82uorescence in situ hybridization. Leukemia 1999;13(1):105\\xe2\\x80\\x939.\\n[23] Fassas AB, Spencer T, Sawyer J, Zangari M, Lee CK, Anaissie E, et al. Both\\nhypodiploidy and deletion of chromosome 13 independently confer poor prognosis in multiple myeloma. Br J Haematol 2002;118(4):1041\\xe2\\x80\\x937.\\n[24] Kaufmann H, Kromer E, Nosslinger T, Weltermann A, Ackermann J, Reisner\\nR, et al. Both chromosome 13 abnormalities by metaphase cytogenetics and\\ndeletion of 13q by interphase FISH only are prognostically relevant in multiple\\nmyeloma. Eur J Haematol 2003;71(3):179\\xe2\\x80\\x9383.\\n[25] Shaughnessy Jr J, Tian E, Sawyer J, McCoy J, Tricot G, Jacobson J, et al. Prognostic\\nimpact of cytogenetic and interphase \\xef\\xac\\x82uorescence in situ hybridizationde\\xef\\xac\\x81ned chromosome 13 deletion in multiple myeloma: early results of total\\ntherapy II. Br J Haematol 2003;120(1):44\\xe2\\x80\\x9352.\\n[26] Dewald GW, Therneau T, Larson D, Lee YK, Fink S, Smoley S, et al. Relationship\\nof patient survival and chromosome anomalies detected in metaphase and/or\\ninterphase cells at diagnosis of myeloma. Blood 2005;106(10):3553\\xe2\\x80\\x938.\\n[27] Chiecchio L, Protheroe RK, Ibrahim AH, Cheung KL, Rudduck C, Dagrada GP,\\net al. Deletion of chromosome 13 detected by conventional cytogenetics is a\\ncritical prognostic factor in myeloma. Leukemia 2006;20(9):1610\\xe2\\x80\\x937.\\n[28] Dohner H, Stilgenbauer S, Benner A, Leupolt E, Krober A, Bullinger L, et al.\\nGenomic aberrations and survival in chronic lymphocytic leukemia. N Engl J\\nMed 2000;343(26):1910\\xe2\\x80\\x936.\\n[29] Rowntree C, Duke V, Panayiotidis P, Kotsi P, Palmisano GL, Hoffbrand AV, et\\nal. Deletion analysis of chromosome 13q14.3 and characterisation of an alternative splice form of LEU1 in B cell chronic lymphocytic leukemia. Leukemia\\n2002;16(7):1267\\xe2\\x80\\x9375.\\n[30] Eis PS, Tam W, Sun L, Chadburn A, Li Z, Gomez MF, et al. Accumulation of\\nmiR-155 and BIC RNA in human B cell lymphomas. Proc Natl Acad Sci USA\\n2005;102(10):3627\\xe2\\x80\\x9332.\\n[31] Staaf J, Lindgren D, Vallon-Christersson J, Isaksson A, Goransson H, Juliusson\\nG, et al. Segmentation-based detection of allelic imbalance and loss-ofheterozygosity in cancer cells using whole genome SNP arrays. Genome Biol\\n2008;9(9):R136.\\n[32] Lai WR, Johnson MD, Kucherlapati R, Park PJ. Comparative analysis of\\nalgorithms for identifying ampli\\xef\\xac\\x81cations and deletions in array CGH data.\\nBioinformatics 2005;21(19):3763\\xe2\\x80\\x9370.\\n[33] Feng S, Liang Q, Kinser RD, Newland K, Guilbaud R. Testing equivalence between\\ntwo laboratories or two methods using paired-sample analysis and interval\\nhypothesis testing. Anal Bioanal Chem 2006;385(5):975\\xe2\\x80\\x9381.\\n\\n\\x0c'"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "text"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "str"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "type(text)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "1312"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "text.find('miR-15')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "-1"
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "text.find('bioanal')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "3467"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "text.find('Bioanal')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "-1"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "text.find('physics')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "2\n"
     ]
    }
   ],
   "source": [
    "str1 = \"this is string example....wow!!!\";\n",
    "str2 = \"is\";\n",
    "\n",
    "print str1.find(str2)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 2",
   "language": "python",
   "name": "python2"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.14"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
